File: modules/01-basic-syntax/typescript/src/modules/references.ts
import * as utils from '../utils/utils';

// References
export function references() {
  utils.print_h3('References');

  let x = 10;
  let ref_x = x;
  ref_x = 20;
  console.log(`x: ${x}`);
  console.log(`ref_x: ${ref_x}`);
  console.log('NOTE: references or pointers do not exists in Typescript');
}

File: modules/01-basic-syntax/python/src/modules/references.py
import src.utils.utils as utils

# References
def references() -> None:
  utils.print_h3(title="References")

  x = 10
  ref_x = x
  ref_x = 20
  print(f"x: {x}")
  print(f"ref_x: {ref_x}")
  print("NOTE: references or pointers do not exists in Python")
